agent/multi_agent.py

```
@@ -9,8 +9,10 @@
 from agent.tools import tools_schema, available_tools
 from database.vector_store import clear_store
 from agent.base_agent import BaseAgent
+from agents import Agent, Runner
 
 client = OpenAI(api_key=settings.OPENAI_API_KEY)
+session_threads = {} # Maps session_id to thread_id
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)